staging/copy.sh

```diff
@@ -218,8 +222,11 @@ if [ "${VERIFYONLY}" = true ]; then
     fi
 fi
 
-echo "move to the client repo"
 # clean the ${CLIENT_REPO}
-ls "${CLIENT_REPO}" | { grep -v '_tmp' || true; } | xargs rm -rf
-mv "${CLIENT_REPO_TEMP}"/* "${CLIENT_REPO}"
+echo "move to the client repo"
+if [ "${DRY_RUN}" = false ]; then
```

Comment:
Could you explain why we need teh DRY_RUN option? When copy.sh is run by verification, it returns after the diff command and never reaches here.